migrations/2023-03-15-173829_create_twabs/up.sql

```diff
@@ -0,0 +1,6 @@
+-- Your SQL goes here
+CREATE TABLE twabs (
+    id VARCHAR PRIMARY KEY,
+    timestamp BIGINT NOT NULL,
+    price DOUBLE PRECISION NOT NULL
```

Comment:
might also be helpful to include the currency the price is denoted in? cc @wilsoncusack if you think this may be helpful as we have different underlyings maybe in the future

Reply:
Good call, it looks like we're not passing `quote_currency` but I assume that would be the source for that field. It doesn't seem to be included in reservoir response.

Reply:
should be included if you decode the payload

Reply:
Interesting, do you know how to do that in rust? Looking at the ethers docs and not having much luck